reconcile/jenkins_webhooks.py

```diff
@@ -24,7 +24,13 @@ def get_hooks_to_add(desired_state, gl):
             current_hooks = gl.get_project_hooks(project_url)
             for h in current_hooks:
                 job_url = h.url
-                trigger = "mr" if h.merge_requests_events else "push"
+                trigger = (
+                    "mr"
+                    if h.merge_requests_events
+                    else "push"
+                    if h.push_events
+                    else "note"
+                )
```

Comment:
now the logic is complex, I can't understand the priority in nested if else, better to extract a function and see if can reuse in `jjb_client` too.